server/routes/taggedBailRoutes.ts

```diff
@@ -289,4 +329,8 @@ export default class TaggedBailRoutes {
     } as Message
     return res.redirect(`/${nomsId}/success?message=${JSON.stringify(message)}`)
   }
+
+  private generateTempId(): string {
+    return `temp${Date.now()}`
```

Comment:
I think we should stick to using UUIDs as the ids.
`randomUUID()`
Instead of checking `temp` in the prefix of the ID, you could add another boolean flag to the SessionAdjustment